src/app/components/manuscript-thumbnails-viewer/manuscript-thumbnails-viewer.component.ts

```diff
@@ -0,0 +1,110 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'evt-manuscript-thumbnails',
+  templateUrl: './manuscript-thumbnails-viewer.component.html',
+  styleUrls: ['./manuscript-thumbnails-viewer.component.scss'],
+})
+
+export class ManuscriptThumbnailsViewerComponent implements OnInit {
+  public grid = [];
+  public col = 2  ;
+  public row = 3;
+  public gridSize = this.col * this.row;
+  public indexPage = 0;
+  private items = [
+    {
+      url: 'https://evt-project.github.io/evt-demo/data/images/single/VB_fol_104v.jpg',
+      name: '104v',
+      active: false,
+    },
+    {
+      url: 'https://evt-project.github.io/evt-demo/data/images/single/VB_fol_104v.jpg',
+      name: '105v',
+      active: false,
+    },
+    {
+      url: 'https://evt-project.github.io/evt-demo/data/images/single/VB_fol_104v.jpg',
+      name: '106v',
+      active: false,
+    },
+    {
+      url: 'https://evt-project.github.io/evt-demo/data/images/single/VB_fol_104v.jpg',
+      name: '107v',
+      active: false,
+    },
+    {
+      url: 'https://evt-project.github.io/evt-demo/data/images/single/VB_fol_104v.jpg',
+      name: '108v',
+      active: false,
+    },
+    {
+      url: 'https://evt-project.github.io/evt-demo/data/images/single/VB_fol_104v.jpg',
+      name: '109v',
+      active: false,
+    },
+    {
+      url: 'https://evt-project.github.io/evt-demo/data/images/single/VB_fol_104v.jpg',
+      name: '110v',
+      active: false,
+    },
+    {
+      url: 'https://evt-project.github.io/evt-demo/data/images/single/VB_fol_104v.jpg',
+      name: '111v',
+      active: false,
+    },
```

Comment:
What about if we have different images or different url? See better how it was handled in the previous version of the code.